app/models/admin_setting.rb

```diff
@@ -53,15 +53,13 @@ class << self
     delegate :default_skin, to: :current
   end
 
-  # run once a day from cron
+  # run hourly with the resque scheduler
   def self.check_queue
-    if self.invite_from_queue_enabled? && InviteRequest.count > 0
-      if Date.today >= self.invite_from_queue_at.to_date
-        new_date = Time.now + self.invite_from_queue_frequency.days
-        self.first.update_attribute(:invite_from_queue_at, new_date)
-        InviteFromQueueJob.perform_now(count: invite_from_queue_number)
-      end
-    end
+    return unless self.invite_from_queue_enabled? && InviteRequest.count.positive? && Time.current >= self.invite_from_queue_at
```

Comment:
Note for future me: Would InviteRequest.any? work here and would it improve performance? Refer to #3360